lib/puppet/provider/pulp_rpmbind/consumer.rb

```diff
@@ -0,0 +1,41 @@
+Puppet::Type.type(:pulp_rpmbind).provide(:consumer) do
+  desc 'Bind/unbind to an RPM repo'
+
+  confine osfamily: :redhat
+  commands consumer: '/bin/pulp-consumer'
+  commands grep:     '/bin/grep'
+
+  def self.instances
+    begin
+      binds = grep('-oP', '^\[\K.*(?=\]$)', '/etc/yum.repos.d/pulp.repo')
```

Comment:
It appears that way.  I was a bit surprised the pulp-consumer binary couldn't list bindings. Admittedly it does seem hacky to grep the pulp.repo file, but it is very simple to do.  The other option is to use the [API](https://docs.pulpproject.org/dev-guide/integration/rest-api/consumer/bind.html) but that would be *way* more complicated and I can't personally see the advantage.  To the contrary, if someone had manually deleted an entry from pulp.repo, the server wouldn't know about it.